index.html

```diff
@@ -302,6 +304,19 @@ <h2 class='mb-0'>Gato-Bots</h2>
                       <circle cx='169' cy='35' r='6' class='fill-base' />
                     </svg>
                   </div>
+                  <div class='carousel-item'>
+                    <svg viewBox='0 0 320 80' width='100%'>
+                      <rect class='fill' fill='#fff' x='140' y='30' width='40' height='40' />
+
+                      <circle cx='151' cy='48' r='7.5' class='accent-stroke' fill='none' stroke-width='2' />
+                      <circle cx='151' cy='48' r='5.5' class='accent-stroke' fill='none' />
+                      <circle cx='169' cy='48' r='7.5' class='accent-stroke' fill='none' stroke-width='2' />
+                      <circle cx='169' cy='48' r='5.5' class='accent-stroke' fill='none' />
+
+                      <path class='fill-base' d='M 145 15 C 140 15, 135 25, 140 30 H 150 C 150 30, 145 25, 145 15 Z' />
+                      <path class='fill-base' d='M 175 15 C 180 15, 185 25, 180 30 H 170 C 170 30, 175 25, 175 15 Z' />
+                    </svg>
+                  </div>
```

Comment:
here's all it takes to add a new head option. Adding a carousel option with an svg element that draws the head base (the `rect`), then the eyes (the `circle`s), then the horns (the `path`s).